### ISSUE TITLE
Options are not stored to localstorage if I browse to another instance 

### ISSUE THREAD

**Issue by localhorst** [howdy-tsc/LemmyTools#5]
So if I change the home instance in the plugin options, it changes on the instance I am currently at. 
But when I visit another instance, it defaults back to the default home instance (i.e., thesimplecorner.org).

I guess the home instance (and other options) need to be stored to a domain-independent storage.

Using: Firefox 114, Mac.

**Comment by howdy-tsc** [comment]
Yep, discovered it too! I have a work around now in the latest. You will just need to edit the script with your home variable for it to stick on remote instances. I'm using the localstorage storage and didn't understand it's scope entirely. Will develop a permanent fix if possible.

**Comment by howdy-tsc** [comment]
This is still in the new version... worked on other functions first. On the todo:
